packages/edit-post/src/components/sidebar/post-status/header/post-last-revision.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { MenuItem } from '@wordpress/components';
+import { addQueryArgs } from '@wordpress/url';
+import { __ } from '@wordpress/i18n';
+
+export default function PostLastRevision() {
+	const { revisionsCount, lastRevisionId } = useSelect(
+		( select ) => ( {
+			revisionsCount:
+				select( editorStore ).getCurrentPostRevisionsCount(),
+			lastRevisionId:
+				select( editorStore ).getCurrentPostLastRevisionId(),
+		} ),
+		[]
+	);
+
+	return (
+		<MenuItem
+			icon={
+				<RevisionsCountBadge>{ revisionsCount }</RevisionsCountBadge>
```

Comment:
This might be helpful as a separate component - `MenuItemBadge` or similar.

Reply:
I'm not sure how widely used this will be yet. There's a maintenance burden involved in adding a new `@wordpress/components` component. What do you think @ciampo?

Reply:
Thanks for the feedback, @ciampo.